.github/workflows/docs.yml

```
@@ -4,7 +4,7 @@ name: Deploy documentation to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["docs-main"]
+    branches: ["main"]
```

```suggestion
    branches: ["docs"]
```
We will still have a docs branch that will allow us to use the latest deployment of feature branches without deling with alpha tags